db/sql/README.md

```diff
@@ -4,4 +4,48 @@ sql files
 This folder contains all SQL files used in the project:
 
 * A folder for each database repository, containing SQL files used by that repository.
-* File [index](./index.js) references and loads all SQL files in the project.
+* File [./config.js](./config.js) provides the tools for loading external SQL files.
+* File [./index.js](./index.js) individually references and loads each SQL file in the project.
+
+## standards
```

Comment:
Prefer capitals for headings. (See other .md files.)

Reply:
I will keep improving this document in future PR-s, and surely will accommodate this case change :wink:

Reply:
@willclarktech this change has been applied and merged ;)